app/assets/stylesheets/ama_layout/layout_components/option-toggle.scss

```diff
@@ -0,0 +1,52 @@
+.option-toggle{
+  &__item{
+    @include clearfix;
+    margin: 1px 0;
+    display: block;
+    height: inherit;
+    line-height: 1;
```

Comment:
Dont think you need display block or height: inherit, possibly not the line-height either.